ocs_ci/ocs/utils.py

```diff
@@ -930,6 +961,8 @@ def run_must_gather(log_dir_path, image, command=None, cluster_config=None):
             f"Failed during must gather logs! Error: {ex}"
             f"Must-Gather Output: {mg_output}"
         )
+        get_logs_ocp_mg_pods()
```

Comment:
Propagate the: log_dir_path to the function as an argument from here.

Reply:
done